.github/workflows/phpunit-mysql.yml

```diff
@@ -32,11 +32,6 @@ concurrency:
   group: phpunit-mysql-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
-env:
-  # Location of the phpunit.xml and phpunit.integration.xml files
-  PHPUNIT_CONFIG: ./tests/phpunit.xml
```

Comment:
Due to the file name mismatch (`tests/unit/phpunit.xml` exists) the tests were never executed.